composer.json

```diff
@@ -21,16 +21,16 @@
     "extra": {
     },
     "require": {
-        "php": "^7.1",
-        "laminas/laminas-eventmanager": "^2.6 || ^3.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3 || ~8.0.0",
+        "laminas/laminas-eventmanager": "^2.6 || ^3.3",
```

Comment:
We can drop the support for laminas-eventmanager v2 here.

Reply:
Right, due to https://github.com/laminas/laminas-eventmanager/pull/10.

Done